src/python/provisioner/provisioner/app.py

```diff
@@ -128,6 +128,6 @@ def provision() -> None:
             username=username,
             cleartext=password,
             role="owner",
-            org="test_org",
+            org="00000000-0000-0000-0000-000000000000",
```

Comment:
is an org_id a tenant_id? if so, can we deduplicate these two names?

Reply:
potentially as a followup PR

Reply:
> is an org_id a tenant_id? if so, can we deduplicate these two names?

Yeah, as far as I know, they are the same.